WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -50,9 +53,9 @@ public static Person fromJSON(JSONObject json, int localTableBlogId) {
             String displayName = json.optString("nice_name");
             String avatarUrl = json.optString("avatar_URL");
             // We don't support multiple roles, so the first role is picked just as it's in Calypso
-            Role role = Role.fromKey(json.optJSONArray("roles").optString(0));
+            String role = json.optJSONArray("roles").optString(0);
```

Comment:
I think I missed this like in the previous PR: the `.optString()` call here can throw a NPE so it's better to use `json.getJSONArray` (instead of `json.optJSONArray`). This way, the potential JSONException will get caught below.